docs/sql-performance-tuning.md

```diff
@@ -158,7 +155,7 @@ broadcast(spark.table("src")).join(spark.table("records"), "key").show()
 {% highlight r %}
 src <- sql("SELECT * FROM src")
 records <- sql("SELECT * FROM records")
-head(join(broadcast(src), records, src$key == records$key))
```

Comment:
did we remove the `broadcast` method?

Reply:
No, the `broadcast` method still available:
https://github.com/apache/spark/blob/18431c7baaba72539603814ef1757650000943d5/R/pkg/R/DataFrame.R#L4129-L4134

Reply:
so what's the recommended API to add hint? `df.hint(...)` or `hint(df, ...)`?

Reply:
hint(df, ...) is for R, there's no df.hint(...) in R？

Reply:
ah nvm, didn't realize it's R code